### PR TITLE
skip reachability test in airgapped

### PR DESCRIPTION
**Pull request by doronyaary-runai**
add a cmd line arg -airgapped and use it to skip reachability tests

### CODE REVIEW

File: internal/cmd/job/internal_cluster_tests.go [hunk 1]
```diff
@@ -61,51 +61,53 @@ func runTestsAndAppendResults(logger *log.Logger) []v2.TestResult {
...
-		})
-	} else {
-		testResults = append(testResults, v2.TestResult{
-			Name:    "RunAI Auth Provider Reachable",
-			Result:  reachable,
-			Message: "",
-		})
-	}
+	airgapped := env.EnvOrDefault(env.AirgappedEnvVar, "false")
+	if airgapped != "true" {
```
Comment (omeryahud) [inline internal/cmd/job/internal_cluster_tests.go:65]: This annoys me for some reason, could you please parse this env var as a boolean and check its value instead?

File: scripts/build-image.sh [hunk 2]
```diff
@@ -1 +1 @@
-docker build --platform x86_64 -f ${DOCKER_FILE} -t ${IMAGE} --build-arg BUILDER_IMAGE=${BUILDER_IMAGE} .
\ No newline at end of file
+docker build --platform linux/amd64 -f ${DOCKER_FILE} -t ${IMAGE} --build-arg BUILDER_IMAGE=${BUILDER_IMAGE} .
```
Comment (omeryahud) [inline scripts/build-image.sh:1]: Isn't this necessary only for you local machine?

Comment (doronyaary-runai) [reply scripts/build-image.sh]: I am not sure how is it compiled in other cases?

Comment (doronyaary-runai) [reply scripts/build-image.sh]: fixed

**Review by omeryahud — APPROVED** [review]

**Pull request merged**